### PR TITLE
added monte carlo integration

### PR DESCRIPTION
**Pull request by Ved-Mahajan**
This is to add the functionality to perform numerical integration using monte carlo integration.

PS: This function is written keeping in mind only 1D integrals.

### CODE REVIEW

File: src/math/monte_carlo_integration.jl [hunk 1]
```diff
@@ -0,0 +1,16 @@
+"""
+Monte carlo integration is a very easy and scalable way to do multidimentional integrals.
+However, only single variable integrals is considered.
+
+Refereces:
+-https://kingaa.github.io/sbied/pfilter/monteCarlo.html
+-https://kingaa.github.io/sbied/pfilter/monteCarlo.html
+
+Contributed by: [Ved Mahajan](https://github.com/Ved-Mahajan)
```
Comment (Panquesito7) [inline src/math/monte_carlo_integration.jl:9]:
```suggestion
Contributed By-: [Ved Mahajan](https://github.com/Ved-Mahajan)
```

File: src/math/monte_carlo_integration.jl [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+"""
+function monte_carlo_integration(f,N,span)
+	x_start,x_end = span
+	rand_array = rand(x_start:0.0001:x_end,N)
+	return sum(f.(rand_array))*(x_end - x_start)/float(N)
+end
+
```
Comment (Panquesito7) [inline src/math/monte_carlo_integration.jl:16]:
```suggestion
```

File: src/math/monte_carlo_integration.jl [hunk 3]
```diff
@@ -0,0 +1,16 @@
+"""
+Monte carlo integration is a very easy and scalable way to do multidimentional integrals.
+However, only single variable integrals is considered.
+
+Refereces:
+-https://kingaa.github.io/sbied/pfilter/monteCarlo.html
+-https://kingaa.github.io/sbied/pfilter/monteCarlo.html
```
Comment (Panquesito7) [inline src/math/monte_carlo_integration.jl:7]:
```suggestion
```

File: src/math/monte_carlo_integration.jl [hunk 4]
```diff
@@ -0,0 +1,16 @@
+"""
+Monte carlo integration is a very easy and scalable way to do multidimentional integrals.
+However, only single variable integrals is considered.
```
Comment (Panquesito7) [inline src/math/monte_carlo_integration.jl:3]:
```suggestion
However, only single variable integrals are considered.
```

File: src/math/monte_carlo_integration.jl [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+Refereces:
+-https://kingaa.github.io/sbied/pfilter/monteCarlo.html
+-https://kingaa.github.io/sbied/pfilter/monteCarlo.html
+
+Contributed by: [Ved Mahajan](https://github.com/Ved-Mahajan)
+"""
+function monte_carlo_integration(f,N,span)
```
Comment (Panquesito7) [inline src/math/monte_carlo_integration.jl:11]: Please add documentation for this function.

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
👍 Good work.

File: src/math/monte_carlo_integration.jl (hunk 5 above)
Comment (Ved-Mahajan) [reply src/math/monte_carlo_integration.jl:26]: done :+1:

Comment (Panquesito7) [reply src/math/monte_carlo_integration.jl:26]: Hmm, I think it'd be better if you do it as the [`knapsack`](https://github.com/TheAlgorithms/Julia/blob/main/src/knapsack/knapsack.jl) problem. I'm not really sure if the Julia documenter is going to work like this. @arubhardwaj?

Comment (Ved-Mahajan) [reply src/math/monte_carlo_integration.jl:26]: What do you think is missing?

IMO, the docs has all that is needed to use that function.

Comment (Azzaare) [reply src/math/monte_carlo_integration.jl:26]: Once again, we should probably merge this and think about the general presentation we want for this repo.

**Review by Azzaare — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]
LGTM. Thanks for your contribution! 👍 🎉

**Pull request merged**